README.md

```diff
@@ -37,30 +37,50 @@ The mosaicking toolset mosaics multiple input images into a set of non-overlappi
 
 Example:
 ```
-python pgc_mosaic.py --pbs --bands 1 --tilesize 20000 20000 --resolution 0.5 0.5 input_dir output_mosaic_name
+python pgc_mosaic.py --slurm --bands 1 --tilesize 20000 20000 --resolution 0.5 0.5 input_dir output_mosaic_name
 ```
 
 This example will evaluate all the 1-band images in input_dir and sort them according to their quality score.  It will submit a job to the cluster queue to build each tile of size 20,000 x 20,000 pixels at 0.5 meters resolution.  The output tiles will be Geotiffs named by appending a row and column identifier to the output_mosaic_name.
 
-### PANSHARPEN
+### pgc_pansharpen
 
-The pansharpening utility applies the orthorectification process to both the pan and multi image in a pair and then pansharpens them using the GDAL tool gdal_pansharpen.  GDAL 2.1 is required for this tool to function.  The --threads flag will apply threading to both gdalwarp and gdal_pansharpen operations.
+The pansharpening utility applies the orthorectification process to both the pan and multi image in a pair and then pansharpens them using the GDAL tool gdal_pansharpen.  GDAL 2.1+ is required for this tool to function.  The --threads flag will apply threading to both gdalwarp and gdal_pansharpen operations.
 
-### NDVI
+### pgc_ndvi
 
 The NDVI utility calculates NDVI from multispectral image(s).  The tool is designed to run on data that have already been run through the pgc_ortho utility.
 
-## INSTALLATION AND DEPENDANCIES
-PGC uses the Mamabaforge installer to build our Python/GDAL software stack.  You can find installers for your OS here:
-https://github.com/conda-forge/miniforge#mambaforge
+## Installation and dependencies
+PGC uses the Miniforge installer to build our Python/GDAL software stack.  You can find installers for your OS here:
+https://github.com/conda-forge/miniforge?tab=readme-ov-file#miniforge3
 
 Users should expect a recent (less than 1-2 years old) version of Python and GDAL to be compatible with tools in this repo.
 The following conda/mamba environment likely contains more dependencies than are needed for tools in this repo, but should suffice:
 ```
-conda create --name pgc -c conda-forge python=3.11 gdal=3.6.4 numpy scipy pandas geopandas rasterio shapely postgresql psycopg2 sqlalchemy configargparse lxml pathlib2 python-dateutil pytest rtree xlsxwriter
+mamba create --name pgc -c conda-forge git python=3.11 gdal=3.6.4 globus-sdk globus-cli numpy scipy pandas geopandas rasterio shapely postgresql psycopg2 sqlalchemy configargparse lxml pathlib2 python-dateutil pytest rtree xlsxwriter tqdm alive-progress pyperclip --yes
```

Comment:
To make this match our current `pgc` environment the following additional changes would be needed:

Bump: `gdal=3.7.2`
Add: `dbfread openpyxl pyperclip xlsxwriter`
Remove: None.

Reply:
Good catch!